lib/index.ts

```diff
@@ -54,43 +57,83 @@ function parseFromFile(
 }
 
 export function buildDependencyTree(
-  /* manifestFile: PaketManifest, */ lockFile: PaketLock, includeDev: boolean = false,
+  manifestFile: PaketDependencies, lockFile: PaketLock, includeDev: boolean = false,
 ) {
   const depTree = {
     dependencies: {},
     name: '',
     version: '',
   } as DepTree;
 
+  const dependencies: { [dep: string]: {} } = {};
+
+  for (const group of manifestFile) {
+    for (const dep of group.dependencies) {
+      const current = dep as NugetDependency;
+      if (current.name) {
```

Comment:
Because only our `NugetDependency` have a `name` attribute. Was wanting to change the types in dependencies vs lockfile parser as they differ in ways that makes this difficult to generalise. They need to be consistent for ease of parsing.